code-challenges.js

```diff
@@ -6,70 +6,255 @@
+    // split up all of the strings in the array
+    let result = []
 
+    // initialize an empty array for later use 
+    for(i = 0; i < spStr.length; i++){
 
-// --------------------6) Create a function that takes in an array of strings and returns the string with the most characters.
-// Expected output: "chopsticks" "stemware"
-
-var utensils = ["fork", "knife", "tongs", "chopsticks", "skewer"]
-var vessels = ["cup", "mug", "stemware", "glass", "tumbler", "tea cup"]
+        // start of for loop
+        if(spStr[i] === "a" || spStr[i] === "e" || spStr[i] === "i" || spStr[i] === "o" || spStr[i] === "u"){ // checks to see if the element in the split up string is a vowel
+            continue; // this continue statement allows the program to ignore the vowels and continue iterating through the string, and with the else statment below, only add the consonants.
+        } else {
+            result.push(spStr[i])
+            // adds every consonant into the result array 
+        }
+    }
+    return result.join("")
+    // returns the joined consonants to pop out a string again 
+}
+console.log(noVowels(true))
+console.log(noVowels(42))
+console.log(noVowels("IAmACodingMaster"))
 
 
 
+// // --------------------6) Create a function that takes in an array of strings and returns the string with the most characters.
+// // Expected output: "chopsticks" "stemware"
 
+var utensils = ["fork", "knife", "tongs", "chopsticks", "skewer"]
+var vessels = ["cup", "mug", "stemware", "glass", "tumbler", "tea cup"]
 
-// --------------------7) Create a function that takes in an array of strings and returns an array with the last letter removed from each string.
-// Expected output: ["Luig", "Peac", "Toa", "Yosh"] ["Ink", "Blink", "Pink", "Clyd"]
+// would like to try and get this one to work 
+const alphaString = (array) => { 
+
+    let maxLength = 0;
+    // setting a variable which will store the max length of a given element within an array
+    let lenArr = [];
+    // initializing an empty array for later use 
+    let ind;
+    // initializing a variable that will set the index of the largest string
+    for(i = 0; i < array.length; i++){
+
+        lenArr.push(array[i].length);
+        // adds the length of the current element to the empty array
+        for(j = 0; j < lenArr.length; j++){
+            // another for loop that will determine the highest number in the array
+            if(lenArr[j] > maxLength){
+                maxLength = lenArr[j]
+                // set maxLength to the element that has been determined larger than the previous value of maxLength (update the value)
+                ind = lenArr.indexOf(maxLength)
+                // set the variable to the index of the largest number 
+            }
+        }
+    }
+    return array[ind]
+}
+console.log(alphaString(vessels))
+console.log(alphaString(utensils))
+
+// // trying a different train of thought that also works 
+const bigString = (array) => {
+
+    let counter = 0;
```

Comment:
```suggestion
    let tracker = 0;
```

`counter` suggests it's tallying a number of items which I think in this case `counter` seems to hold the length of the longest word encountered

Reply:
awesome practicing different ways of doing the same thing!!